src/rest_vol_dataset.c

```diff
@@ -288,7 +288,7 @@ RV_dataset_open(void *obj, const H5VL_loc_params_t *loc_params, const char *name
     RV_object_t *parent = (RV_object_t *) obj;
     RV_object_t *dataset = NULL;
     H5I_type_t   obj_type = H5I_UNINIT;
-    htri_t       search_ret;
+    hbool_t       search_ret;
```

Comment:
Four remaining `hbool_t` -> `htri_t` reversions for this file, including `search_ret` and `is_variable_str`